### PR TITLE
DOC-75: Polish Users JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namepaces:
- Users

Case: https://highfidelity.atlassian.net/browse/DOC-75

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19772/HighFidelity-Beta-PR15678-0bdca38.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19772/HighFidelity-Quest-Beta-PR15678-0bdca38.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19772/HighFidelity-Beta-PR15678-0bdca38.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19772/HighFidelity-Beta-Interface-PR15678-0bdca38.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19772/HighFidelity-Beta-PR15678-0bdca38.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19772/HighFidelity-Beta-Interface-PR15678-0bdca38.exe

File: libraries/script-engine/src/UsersScriptingInterface.h [hunk 1]
```diff
@@ -49,130 +51,161 @@ class UsersScriptingInterface : public QObject, public Dependency {
...
      * @function Users.getCanKick
-     * @returns {boolean} <code>true</code> if the domain server allows the client to kick (ban) other users, otherwise 
+     * @returns {boolean} <code>true</code> if the domain server allows the client to kick and ban users, otherwise 
      *     <code>false</code>.
      */
     bool getCanKick();
 
     /**jsdoc
-     * Toggle the state of the space bubble feature.
+     * Toggles the state of the privacy bubble.
```
Comment (jennaingersoll) [inline libraries/script-engine/src/UsersScriptingInterface.h]: "privacy bubble" is now called "shield".

Comment (ctrlaltdavid) [reply libraries/script-engine/src/UsersScriptingInterface.h]: Changed all occurrences to "privacy shield".

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19782/HighFidelity-Beta-PR15678-eac7d29.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19782/HighFidelity-Beta-Interface-PR15678-eac7d29.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19782/HighFidelity-Beta-PR15678-eac7d29.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19782/HighFidelity-Quest-Beta-PR15678-eac7d29.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19782/HighFidelity-Beta-PR15678-eac7d29.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19782/HighFidelity-Beta-Interface-PR15678-eac7d29.exe

**Pull request merged**